exp/views/study.py

```diff
@@ -276,3 +300,10 @@ def get_context_data(self, **kwargs):
             }, indent=4) for resp in context['responses']]
         context['all_responses'] = ', '.join(context['response_data'])
         return context
+
+
+class PreviewProxyView(ProxyView, LoginRequiredMixin):
+    upstream = settings.EXPERIMENT_BASE_URL
+
+    def dispatch(self, request, path, *args, **kwargs):
```

Comment:
I didn't have it there at first, but put it in to avoid `dispatch() got an unexpected keyword argument 'uuid'` errors.